services/thanos/12.4.3/defaults/cm.yaml

```diff
@@ -0,0 +1,64 @@
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: thanos-12.4.3-d2iq-defaults
+  namespace: ${releaseNamespace}
+data:
+  values.yaml: |
+    ---
+    image:
```

Comment:
The default here is [using docker hub](https://github.com/bitnami/charts/blob/main/bitnami/thanos/values.yaml#L54-L57).
But since I am overriding the image does this mean I also need to change the chart bump tool? @gracedo @bcmendoza ?

Reply:
hmm. can we use the default image instead of overriding?

Reply:
we can. just 1 more image competes for dockerhub limit. but guess thats ok since customer can always override themselves if they are really bothered by this.